packages/types/src/app-instance.ts

```diff
@@ -10,7 +10,7 @@ export enum AssetType {
 export interface Terms {
   assetType: AssetType;
   limit: BigNumber;
-  token: Address;
+  token?: Address;
```

Comment:
```suggestion
  token: Address;
```

Reply:
What's the address of the token if the assetType is ETH?

Reply:
It is the default value for the `address` data type which is `AddressZero`